CHANGELOG.md

```diff
@@ -1,11 +1,15 @@
 # Changelog
 
-### 24.9.5 [#771](https://github.com/openfisca/openfisca-core/pull/771)
+### 24.9.7 [#769](https://github.com/openfisca/openfisca-core/pull/769)
+
+- Unify the protocol for appending sub-parameters
```

Comment:
Not sure this is limpid for a user of the OpenFisca-Core library. 
What does mean "appending sub-parameters" from them? Is there a _protocol_ for that? 

From the user point of view, I think what has been done here is:

```
- Fix an inconsistency in parameter navigation
   - In some cases, for instance while using extensions, the `parameters.path.to.subparams` notation was not working
```

On a meta note, I think this is another reason why the Changelog should ideally written before the review, so that it can be improve, as it's a pretty important part of each PR.

Reply:
Yes, too implementation-centric, thanks for the suggestion; I can pick it up on a later PR. I didn't much like "fix" as a verb though, it's too generic, and that's part of why I looked for more specific terms. How about

- Ensure `path.to.parameter` syntax works consistently across country and extension

On the meta level: fixing unnecessary merge conflicts is one of the least appealing parts of the merge process at the moment, so much I'd make the diametrically opposed policy proposal, i.e. delay the CHANGELOG.md/setup.py until just before merge.

As a "best of both worlds" proposal, I suggest improving the description collaboratively by editing the PR's description at review time. Then creating the CHANGELOG entry is a matter of copy/paste (until we automate it away).

Reply:
(Never mind, I hadn't seen the fixup PR.)

Reply:
We can include the fixup PR content in another PR to come (especially as right now editing the changelog triggers the version bump requirement), and feel free to edit the text it if you like.